sunpy/__init__.py

```diff
@@ -16,26 +16,13 @@
 from sunpy.util.config import load_config, print_config
 from sunpy.util.logger import _init_log
 
-# Enforce Python version check during package import.
-__minimum_python_version__ = "3.6"
-
 try:
     from .version import __version__
 except ImportError:
     print("version.py not found, please reinstall sunpy.")
     __version__ = "unknown"
 
 
-class UnsupportedPythonError(Exception):
-    pass
-
-
-if sys.version_info < tuple(int(val) for val in __minimum_python_version__.split('.')):
```

Comment:
Is there still a check for python version somewhere that gives a nice error?

Reply:
I suppose not at runtime, I will put this back.